src/styles/_catalog.scss

```diff
@@ -1,10 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  gap: 48px;
+  justify-content: center;
   max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
Use variable for with card

Reply:
It seems, you meant "width" ) it is done )